test/helpers/replicate.js

```diff
@@ -36,6 +36,7 @@ module.exports = async function replicate(person1, person2, opts = {}) {
     pull.unique(),
     (drain1 = pull.drain((feedId) => {
       person1.ebt.request(feedId, true)
+      person2.ebt.request(feedId, true)
```

Comment:
remove this file and switch to the new replicate function? not much difference in the end i think but :shrug: